channels/cl.m3u

```diff
@@ -32,7 +32,15 @@ https://unlimited6-cl.dps.live/carolinatv/carolinatv.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="ElPinguinoTV.cl" tvg-name="El Pingüino TV" tvg-country="CL" tvg-language="Spanish" tvg-logo="https://i.imgur.com/3ob9ss3.jpg" group-title="",El Pingüino TV (240p) [Not 24/7]
 http://167.114.133.205:1935/live/pinguinotv/playlist.m3u8
 #EXTINF:-1 tvg-id="ElPinguinoTV.cl" tvg-name="El Pingüino TV" tvg-country="CL" tvg-language="Spanish" tvg-logo="https://i.imgur.com/3ob9ss3.jpg" group-title="",El Pingüino TV (360p) [Not 24/7]
-http://167.114.133.205:1935/live/pinguinotv_360p/playlist.m3u8
+http://167.114.133.205:1935/live/pinguinotv_360p/playlist.m3u8#EXTINF:-1 tvg-id="TVOSanVicente.cl" tvg-name="TVO San Vicente" tvg-country="CL" tvg-language="Spanish" tvg-logo="https://i.imgur.com/fvnO1X9.png" group-title="",TVO San Vicente
+https://unlimited2-cl-movistar.dps.live/tvo/tvo.smil/tvo/livestream1/chunks.m3u8
+#EXTINF:-1 tvg-id="Canal9Concepción.cl" tvg-name="Canal 9" tvg-country="CL" tvg-language="Spanish" tvg-logo="https://i.imgur.com/UnWMRrT.png" group-title="",Canal 9
+https://unlimited2-cl.dps.live/c9/c9.smil/c9/livestream2/chunks.m3u8
+#EXTINF:-1 tvg-id="SantiagoTV.cl" tvg-name="Santiago TV" tvg-country="CL" tvg-language="Spanish" tvg-logo="https://i.imgur.com/9K9lQl6.jpg" group-title="",Santiago TV
+https://stv.janus.cl/playlist/stream.m3u8
+#EXTINF:-1 tvg-id="CNNChile.cl" tvg-name="CNN Chile" tvg-country="CL" tvg-language="Spanish" tvg-logo="" group-title="",CNN Chile
```

Comment:
We sort playlists according to their main headquarters origin, CNN goes to the US playlist